docs/reference/tab-widgets/inference-api/infer-api-ingest-pipeline-widget.asciidoc

```diff
@@ -66,8 +72,8 @@ include::infer-api-ingest-pipeline.asciidoc[tag=openai]
   </div>
   <div tabindex="0"
        role="tabpanel"
-       id="infer-api-ingest-azure-openai-tab"
-       aria-labelledby="infer-api-ingest-azure-openai"
+       id="infer-api-ingest-azure-ai-studio-tab"
+       aria-labelledby="infer-api-ingest-azure-ai-studio"
```

Comment:
It is still not quite right. Now we miss the `Azure Open AI` tab. The following buttons and tabs need to be in this file:
* Cohere
* HuggingFace
* OpenAI
* Azure OpenAI (we miss this one now, my suggestion adds it back)
* Azure AI Studio
* Mistral
The buttons are great! The issue is in the second half of the file where we configure the tabs' content. We need a section that references the `azure-openai-tab` and includes the corresponding asciidoc file. Right now, this section from line 72 to line 82 contains references to `azure-ai-studio` but the include file (line 80) refers to `azure openai`. The suggestion solves that problem. However, we still need a section for `azure-ai-studio`. I know, the names are pretty confusing. :) 

```suggestion
       id="infer-api-ingest-azure-openai-tab"
       aria-labelledby="infer-api-ingest-azure-openai"
```

Reply:
Ohhhh - sorry! I saw that and thought to myself "that doesn't look right", so I intuitively changed it. Changing it back now. Thanks for the explanation.

Reply:
Oh wait - hold up, there is a section in the infer-api-ingest-pipeline file for[ azure-ai-studio](https://github.com/elastic/elasticsearch/blob/fdb5058b13abbbc5ce3a5d7c0ab1a7878c72176c/docs/reference/tab-widgets/inference-api/infer-api-ingest-pipeline.asciidoc?plain=1#L116) -- is this what you think is missing here? If so, I think it's just a matter of making sure the `include::` references this (which, it does not at the moment...) let me try that and commit it and see...

Reply:
@szabosteve - I _think_ my latest commit did indeed fix it properly (without the workaround) - let me know your thoughts!

Reply:
(although it's not working properly for the reindexing steps examples... debugging this now)

Reply:
@markjhoy The `azure-openai` tab content is still missing. After `openai` we need an `azure-openai` but now we just have an `azure-ai-studio`. To fix this, we need to add the following code after line 71:

```
</div>
  <div tabindex="0"
       role="tabpanel"
       id="infer-api-ingest-azure-openai-tab"
       aria-labelledby="infer-api-ingest-azure-openai"
       hidden="">
++++
include::infer-api-ingest-pipeline.asciidoc[tag=azure-openai]
++++
```

Reply:
> The azure-openai tab content is still missing

Yep - I found that and I _think_ my latest commit should fix it (also, the "hidden" was missing from the HuggingFace portion for the "Create an inference endpoint" part - which caused it to be visible on first load as well... - fixed this in the commit too)... after the build is complete, hopefully all will be working as it should 🤞

Reply:
... and so close... Mistral is missing from the index pipeline step, but other than that, everything else is working... another round incoming shortly...